content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -0,0 +1,169 @@
+    105.133.10.12
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output as you follow the task, retry after a few seconds.
+Caching and propagation overhead can cause some delay.
+{{< /warning >}}
+
+## IP-based allow list and deny list
+
+1. The following command creates the authorization policy, `ingress-policy`, for
+the Istio ingress gateway. The following policy sets the `action` field to `ALLOW` to
+allow the IP addresses specified in the `ipBlocks` to access the ingress gateway,
+IP addresses not in the list will be denied, also known as whitelist-style access control.
+The `ipBlocks` supports both single IP address and CIDR notation.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
+    EOF
+    {{< /text >}}
+
+1. Verify that a request to the ingress gateway is denied.
+
+    {{< text bash >}}
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    403
+    {{< /text >}}
+
+1. Update the `ingress-policy` to include your client IP address.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           ipBlocks: ["1.2.3.4", "5.6.7.0/24", "$CLIENT_IP"]
+    EOF
+    {{< /text >}}
+
+1. Verify that a request to the ingress gateway is allowed.
+
+    {{< text bash >}}
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+1. The following command updates the `ingress-policy` authorization policy to set
+the `action` key to `DENY` so that the IP addresses specified in the `ipBlocks` are
+not allowed to access the ingress gateway, also known as blacklist-style access control:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: DENY
+      rules:
+      - from:
+        - source:
+           ipBlocks: ["$CLIENT_IP"]
+    EOF
+    {{< /text >}}
+
+1. Verify that a request to the ingress gateway is denied.
+
+    {{< text bash >}}
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    403
+    {{< /text >}}
+
+1. You could use an online proxy service to access the ingress gateway that uses a
```

Comment:
Speculative action requested - imo - should not be in the documentation. Either indicate a direct action in a command block (and change `You could` ->`Use`).

Change `is allowed.` -> `is allowed:` followed by an example.

If that doesn't work, please convert to a callout:
https://istio.io/about/contribute/creating-and-editing-pages/#callouts